src/design-system/typography/typeHierarchy.ts

```diff
@@ -57,18 +75,36 @@ export const typeHierarchy = {
         ios: -0.3,
       },
     },
+    '12px': {
+      fontSize: 12,
+      letterSpacing: 0.6,
+      lineHeight: 14,
+      marginCorrection: {
+        android: -0.3,
+        ios: -0.3,
+      },
+    },
     '14px': {
       fontSize: 14,
       letterSpacing: 0.6,
-      lineHeight: 17,
+      lineHeight: 19,
       marginCorrection: {
         android: -0.1,
         ios: -0.3,
       },
     },
+    '15px': {
```

Comment:
This needs an example in `Text.examples.tsx`.

Reply:
added